spec/unit/views/components/paginated_collection_spec.rb

```diff
@@ -198,9 +198,10 @@ def paginated_collection(*args)
       end
 
       describe "set to false" do
-        let(:pagination) { paginated_collection(collection, pagination_total: false) }
-
         it "should not show the total item counts" do
+          expect(collection).not_to receive(:num_pages)
+          expect(collection).not_to receive(:total_pages)
+          pagination = paginated_collection(collection, pagination_total: false)
```

Comment:
This spec doesn't assert on COUNT queries. If this was ever fixed, it seems like a regression was added: 
https://github.com/activeadmin/activeadmin/pull/5388